src/bundle/Resources/public/scss/_contentbox.scss

```diff
@@ -0,0 +1,77 @@
+.ez-contentbox {
+    width: 100%;
+
+    .ez-list-toolbar {
+        display: flex;
+        align-items: center;
+        padding: 0.5em 0;
+        background: $ez-color-base-pale;
+        border-radius: 4px 4px 0 0;
+        font-size: 1rem;
+
+        .ez-list-toolbar-label {
```

Comment:
we use BEM so this should be `.ez-list-toolbar__label, with sass you can write: `&__label`.